qiskit/primitives/base/base_estimator.py

```diff
@@ -254,3 +325,43 @@ def parameters(self) -> tuple[ParameterView, ...]:
             Parameters, where ``parameters[i][j]`` is the j-th parameter of the i-th circuit.
         """
         return tuple(self._parameters)
+
+
+BaseEstimator = BaseEstimatorV1
+
+
+class BaseEstimatorV2:
+    """Estimator base class version 2.
+
+    An estimator estimates expectation values for provided quantum circuit and
+    observable combinations.
+    """
+
+    def __init__(self, precision: float | None):
+        """
+        Args:
+            precision: a target precision of mean expectation value estimates.
```

Comment:
To me this is clearer without the word "mean"

```suggestion
            precision: A target precision for expectation value estimates.
```